www/res/scripts/luaToPhp.php

```diff
@@ -31,6 +31,14 @@ function trimval(string)
 function trimval($str)
 {
   $str = trim($str);
+
+  // Evaluated numbers used in fire rates like '10/10'
+  if (preg_match('/^((?:[0-9\.]+ *[\/+-] *)*[0-9\.]+),?$/', $str, $matches))
+  {
+    eval('$str = ' . $matches[1] . ';');
+    return (float)$str == (int)$str ? (int)$str : (float)$str;
+  }

```

Comment:
Usually any eval rings all alarm bells. But with that regex it seems safe.

Yet, it seems safer and more readable to go with `\w*` over ` *`

Reply:
`\w*` is valid lua, so I agree that it is better.